src/components/Modeler.vue

```diff
@@ -336,10 +336,20 @@ export default {
       const registeredIndex = node.inspectorConfig[0].items.findIndex(item => {
         return config.id && config.id === item.id;
       });
+        
+      let items;
+      if (config.container) {
+        // if it is a container, don't put it inside the default container
+        items = node.inspectorConfig[0].items;
+      } else {
+        // if it is not a container, put it in the default container
+        items = node.inspectorConfig[0].items[0].items;
+      }
+
       if (registeredIndex === -1) {
-        node.inspectorConfig[0].items[0].items.push(config);
+        items.push(config);
       } else {
-        node.inspectorConfig[0].items[0].items[registeredIndex] = config;
+        items[registeredIndex] = config;
```

Comment:
I think this whole function is a little hard to read and can be refactored. There's a few things I've noticed, and you can let me know what you think :)

Whether or not `config` is a container, they both work with a common section of the `node` that's being modified: `node.inspectorConfig[0].items`. This path is going to exist and be the same for every node, and is the list of accordions for each node (currently every node only has one "default" accordion container, "Configuration"). This could be assigned to a variable to make it easier to read.